pydarn/plotting/rtp.py

```diff
@@ -6,13 +6,16 @@
 """
 Range-time Intensity plots
 """
-import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib import dates, colors, cm, ticker
+import numpy as np
 from typing import List
-from datetime import datetime
+from datetime import datetime, timedelta
 
-from pydarn import SuperDARNRadars
+from pydarn import DmapArray, DmapScalar, utils, rtp_exceptions, SuperDARNCpids
```

Comment:
Dmap, which stands for DataMap (thanks Marina!) should be DMap throughout.

Reply:
Started changing DMAP, dmap, Dmap to DMap. For the class names, I won't change it because of the naming standards PEP8 suggests.

Reply:
Complying with PEP8 for variable names and class names is definitely the way to go.